README.md

```diff
@@ -99,6 +99,8 @@ More examples could be found at [go-storage-example](https://github.com/beyondst
 - [ipfs](https://github.com/beyondstorage/go-service-ipfs): [InterPlanetary File System](https://ipfs.io)
 - [memory](https://github.com/beyondstorage/go-service-memory): data that only in memory
 - [minio](https://github.com/beyondstorage/go-service-minio): [MinIO](https://min.io)
+- [bos](https://github.com/beyondstorage/go-service-bos): [Baidu Object Storage](https://cloud.baidu.com/product/bos.html)
```

Comment:
How about sort these services in order?

Reply:
What are the rules I should follow to sort these services?Alphabetical order?

Reply:
Yep, alphabetical order.